README.md

```diff
@@ -11,3 +11,16 @@ This Repository
 ===============
 
 This repository contains the markup needed to deploy the floeproject.org website.
+
+
+News Item Template
+==================
+
+An html template is provided for the news item in `./news/YYYY-MM-DD-news-item-title.html`
+Save your renamed file to the `./news` folder.
+
+Note!
+=====
+1. You will also need to add your news item to the top of the news archive (in `./news/index.html`).
+2. Remember to update the floeproject.org page with a short summary of your news item.
```

Comment:
Change this:
Remember to update the floeproject.org page

To this:
Remember to update the floeproject.org \`./index.html\` page...